models/articles.models.js

```diff
@@ -0,0 +1,29 @@
+const connection = require("../db/connection");
+const { articleData } = require("../db/data/test-data");
+
+
+exports.getArticle = (articleId) => {
+  const validStringItems = ["1", "2", "3", "4", "5", "6", "7", "8", "9"];
```

Comment:
I understand what you're going for here, but this isn't the way to do it. PSQL will throw a very specific error code if handed an id that is not a digit. You don't need to validate it, just create an error handler for it. This is covered in the standard error handling notes for back end.

If the number is valid and the db returns a row count of 0, that is the time for a custom 404 promise reject.